### ISSUE TITLE
2.1.0_1 and 2.1.1 gives an error when HEAD FFmpeg is used

### ISSUE THREAD

**Issue by retokromer** [mediamicroservices/mm#74]
```
Installing dependencies for mediamicroservices/mm/mm: ffmpeg
Error: /usr/local/Cellar/ffmpeg/3.1.4 is not a directory
```

**Comment by dericed** [comment]
oops, @privatezero was right all along

**Comment by kieranjol** [comment]
Is this something to do with mpv requiring a specific version of ffmpeg as a dependency?

**Comment by retokromer** [comment]
Sorry, after a quick look, I cannot see where I should modify in order to accept again both situations the last release and the current HEAD.

**Comment by retokromer** [comment]
FYI: Broken also under Linuxbrew (where the file structure is different).

**Comment by retokromer** [comment]
FYI 2: I do not longer delve deeper into this issue, in order to avoid overlapping like https://github.com/mediamicroservices/homebrew-mm/pull/2

**Comment by retokromer** [comment]
I am becoming crazy: on our Linuxes (Ubuntu and Slackware) 2.1.1 works fine now with HEAD, but not on macOS, even after two `brew update -force`. Life is hard!

**Comment by dericed** [comment]
please test after https://github.com/mediamicroservices/homebrew-mm/commit/983ba82085ff4c94cfcbdd89e153b2b3aabe82f8

**Comment by retokromer** [comment]
Oops! My comments shows only there https://github.com/mediamicroservices/homebrew-mm/commit/983ba82085ff4c94cfcbdd89e153b2b3aabe82f8

**Comment by retokromer** [comment]
It seems to me, this is an FFmpeg issue. FFmpeg is not installing anymore `ffplay` when using HEAD. The use of the `--with-ffplay` parameter is deprecated, but the command `ffplay` itself should still be available. If @dericed or others can confirm, I will close this issue.

**Comment by dericed** [comment]
IIUC `--with-ffplay` is replaced with `--with-sdl`.

**Comment by retokromer** [comment]
Yes, but I didn’t hear that `ffplay` is dumped. The discussion was about `ffserver`.

**Comment by retokromer** [comment]
FYI, @dericed : Now HEAD works if both `--with-sdl --with-sdl2` are set.

**Comment by dericed** [comment]
is there any reason not to change https://github.com/mediamicroservices/homebrew-mm/blob/master/mm.rb#L14 to use `--with-sdl --with-sdl2`?

**Comment by retokromer** [comment]
The version 3.1.5 should be out in the hext hours. This (as well as 3.1.4) works fine. I guess the double `sdl` is a mistake which should be fixed at HEAD level. Perhaps tomorrow I can delve into this (depending on the attendance at the Home Movie Day in Tirana).

**Comment by retokromer** [comment]
`--with-ffplay` is available again in version 3.2.
